src/FixerDescriptionAwareInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer;
+
+/**
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ */
+interface FixerDescriptionAwareInterface
+{
+    /**
+     * @return FixerDescription
+     */
+    public function getFixerDescription();
```

Comment:
> i think having one interface to deal with all type of descriptions of a fixer would be better

let us put summary of fixer into DTO, let it be.

Then, if you remove `getDescription: string` from main interface and make `getDefinition` optional, you will end up with fixers without any description. Clean for code, but not sure if it's good for UX